backend/app/api/v1/endpoints/cpt/maps/telco.py

```diff
@@ -14,8 +14,15 @@ async def telcoMapper(
     start_datetime: date, end_datetime: date, size: int, offset: int, filter: str
 ):
     updated_filter = await get_updated_telco_filter(filter)
+    sort = None
```

Comment:
I won't object to the "constant value variable" here because it does have the advantage of providing a name for each of the `getData` parameters... however a more "Pythonic" way to do that would be to use keyword parameters with `sort=None`. (And using keyword arguments here would have avoided what was apparently a bug that should be breaking the home page Telco mapper right now...)